src/main/resources/db-scripts/clickhouse/clickhouse.sql

```diff
@@ -54,30 +54,82 @@ WHERE gene.entrez_gene_id > 0 AND gene.type = 'protein-coding';
 
 CREATE TABLE sample_derived
 (
-    sample_unique_id         String,
-    sample_unique_id_base64  String,
-    sample_stable_id         String,
-    patient_unique_id        String,
-    patient_unique_id_base64 String,
-    patient_stable_id        String,
-    cancer_study_identifier LowCardinality(String),
-    internal_id             Int
+    sample_unique_id            String,
+    sample_unique_id_base64     String,
+    sample_stable_id            String,
+    patient_unique_id           String,
+    patient_unique_id_base64    String,
+    patient_stable_id           String,
+    cancer_study_identifier     LowCardinality(String),
+    internal_id                 Int,
+    -- fields below are needed for the DETAILED projection
+    sequenced                   Int,
+    copy_number_segment_present Int,
+    patient_internal_id         Int,
+    sample_type                 String,
+    cancer_study_id             Int,
+    type_of_cancer_id           String,
+    cancer_study_name           String,
+    cancer_study_description    String,
+    cancer_study_public         Int,
+    cancer_study_pmid           String,
+    cancer_study_citation       String,
+    cancer_study_groups         String,
+    cancer_study_status         Int,
+    cancer_study_import_date    DateTime,
+    reference_genome            String
```

Comment:
Also, would be curious how much speed we get with the wide column vs just doing a simple join or using a dictionary. Don't want to just randomly add more columns to these tables if they aren't needed

Reply:
Just tried moving everything, including the `sequenced` and `copy_number_segment_present` logic, to the SQL layer. It doesn't seem to degrade the performance significantly. The wide table is slightly faster but probably negligible.

Reply:
Reduced the number of columns by https://github.com/cBioPortal/cbioportal/pull/11393/commits/34f3b7748139f24cdd76aaff9dd731a40fac6553.

For now keeping `sequenced` and `copy_number_segment_present` as well as `sample_type` and `patient_internal_id`, but it is also possible to derive them in SQL if we don't want to add any new columns to `sample_derived` at all.